build.sbt

```diff
@@ -0,0 +1,58 @@
+import Dependencies._
+
+lazy val spark = "2.0.2"
+lazy val si = spark match {
```

Comment:
I can't use destructuring patterns here because of SBT limitations, so I just bind `si` and reference its fields below.